src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 spring.social.twitter.appId=DJ1lZnxVadvGsTqiFhPs0jasq
-spring.social.twitter.appSecret=PWxJ449wRhn1II1QQQ8tZjnCiwYDyTErkt8FOIukxmMKwqEMpw
\ No newline at end of file
+spring.social.twitter.appSecret=PWxJ449wRhn1II1QQQ8tZjnCiwYDyTErkt8FOIukxmMKwqEMpw
```

Comment:
Credentials don't belong on GitHub at all. You should add these to the .gitignore.

If you think about this, I could now go and use your credentials and play around with your account.

Reply:
Oh. That is a very stupid mistake. I will add it to .gitignore.
I think there should be 2 application.properties files (ofcourse with different names); one having the credentials and the other having the other things like port number and that jackson indentation. I am not sure though that it is possible to have to files for application.properties instead of one.